.github/workflows/test_examples.yml

```diff
@@ -14,12 +14,11 @@ jobs:
       matrix:
         platform: [ubuntu-latest, macos-latest]
         # after fix of https://github.com/near/nearcore/pull/9140#issuecomment-1576655073, it should be [stable]
-        toolchain: [1.69.0]
+        toolchain: [1.70.0]
```

Comment:
@frol should it be stable according to comment on 16?

Reply:
Change to stable. Thanks